### ISSUE TITLE
Can't send requests to Auth Analyzer from Organizer tab.

### ISSUE THREAD

**Issue by Tib3rius** [simioni87/auth_analyzer#47]
For some reason, even though I can send requests to Auth Analyzer from the Proxy History, if I select requests in the Organizer tab, I cannot do the same from there.

Hopefully should be a quick fix, this would be a great addition!

**Comment by simioni87** [comment]
Good catch, thanks! You're more than welcome to dig into it, fix it, and commit if you find a quick solution.
